### ISSUE TITLE
Scoping the website and repositories

### ISSUE THREAD

**Issue by cforgaci** [Rbanism/Rbanism#1]
While adding content to the website for the reproducibility part, I realised that the entire Rbanism project could be hosted there, with the different stages being added gradually as they occur. That way, at the end of the fellowship, everything would be in the same place. It is very easy to add and structure new sections with Quarto.

Now, the website is rendered from the ReproHack repository, which means that it does not make sense that URL ends with ReproHack. Should we rename this repository with a more meaningful name like Rbanism? 

The topics of the fellowship—reproducibility (& automation), scalability—could then have separate sections on the website, including the resources/results of the workshops and challenges and linking to corresponding repositories. For the first workshop, I would set up a Carpentries website in a separate repository. For the challenges, guidelines and receiving submissions would be accommodated in separate repos as well.

In short, I am thinking about:
- a website repo called **Rbanism**, linking to relevant content from all the other repos and mostly meant to show resources and results
- a Carpentry-style reproducibility & automation workshop repo/website called **reproducibility-workshop**
- a (Carpentry-style) scalability repo/website called **scalability-workshop**
- two repos for the two challenges, with instructions added in the README file, called **ReproHack** and **ScaleUp** respectively
- in addition, I will also host the Geospatial DC website in this organisation

@bvreede, what do you think?

**Comment by bvreede** [comment]
This sounds like a good idea. The current repository can easily be renamed, and as the need for new repositories arises, they can be added under the same organisation. Go for it!
